views/features.erb

```diff
@@ -23,12 +23,16 @@
 				<li>It’s resilient since it operates across three independent availability zones</li>
 				<li>It’s built on an open source PaaS platform (Cloud Foundry)</li>
 				<li>You can decide to move to other Cloud Foundry-based PaaS providers, including on GCloud, as you’re not locked into our platform</li>
+				<li>It's best suited for <a href="https://12factor.net/">Twelve-Factor App development</a></li>
 			</ul>
 			<h2 class="heading-medium">Security</h2>
 			<ul class="list list-bullet">
 				<li>It’s accredited for handling personal and confidential information classified as Official</li>
 				<li>It allows SSH access to containers</li>
 				<li>Applications running on the platform are isolated from each other so they can’t read or change each other's code, data or logs</li>
+				<li>Modern TLS standards are enforced for all traffic entering the platform</li>
+				<li>Applications can communicate internally via a private network</li>
+				<li>Traffic inside the platform is protected by industry standard, strong IPsec encryption</li>
```

Comment:
I think the juxtaposition of these 2 lines (private network + IPSec) is misleading. IPSec protects traffic after TLS is unwrapped, i.e. from router to cell. I'm not sure that IPSec is enabled between cells - @alext could you clarify?

Reply:
@LeePorte might also have context about IPSec in a cell-to-cell context.

Reply:
We're running IPSec on the cells too https://github.com/alphagov/paas-cf/blob/master/manifests/cf-manifest/operations.d/530-ipsec.yml

Reply:
Traffic between cells is not currently under TLS though

Reply:
We've /got/ to have IPSec configured on the cells, otherwise the return traffic to the routers wouldn't work. I'm wondering if a new connection from a cell to another cell (over Istio, as I understand it's passing from some pivotal.io blog post) involves IPSec.

Reply:
@LeePorte Confirmed that cell-to-cell is [also IPsec'd](https://github.com/alphagov/paas-cf/blob/master/manifests/cf-manifest/operations.d/530-ipsec.yml)

Reply:
@LeePorte actually, reading that opsfile, I think it's clear that if that's only place we configure IPSec, then we *don't* have it for cell-to-cell traffic. /cc @tnwhitwell

Reply:
As far as I know IPSec is only enabled for router <-> cell traffic. It's not enabled for cell <-> cell.